integration_test/workflow_tests/test_packages/catalog/test_provider_does_not_exist.py

```diff
@@ -1,11 +1,13 @@
 import json
-import logging
 import time
 import uuid
 from unittest import TestCase
 
 import helpers
+from custom_logger import CustomLoggerAdapter
 
+#Set the logger
+logging = CustomLoggerAdapter.set_logger("Ingest TestProviderDoesNotExist")
```

Comment:
I might fiddle with the [__name__](https://www.tutorialspoint.com/how-do-i-find-the-current-module-name-in-python#:~:text=A%20module%20can%20find%20out,is%20running%20as%20a%20script.) stuff to see if we can fully automate the parameter for set_logger. Not needed for this card though.

Reply:
Sounds good.